faq/faq.rst

```diff
@@ -59,13 +52,29 @@ Other queries which, by contrast, combine data from multiple nodes, do not suppo
 * Set operations
 * Transactional semantics for queries that span across multiple shards
 
+.. _faq_choose_shard_count:
+
 How do I choose the shard count when I hash-partition my data?
 --------------------------------------------------------------
-.. _faq_choose_shard_count:
 
-Optimal shard count is related to the total number of cores on the workers. Citus partitions an incoming query into its fragment queries which run on individual worker shards. Hence, the degree of parallelism for each query is governed by the number of shards the query hits. To ensure maximum parallelism, you should create enough shards on each node such that there is at least one shard per CPU core.
+One of the choices when first distributing a table is its shard count. This setting can be set differently for each table, and the optimal value depends on use-case. It is possible, but difficult, to change the count after cluster creation, so use these guidelines to choose the right size.
+
+In the :ref:`mt_blurb` use-case we recommend choosing between 32 - 128 shards.  For smaller workloads say <100GB, you could start with 32 shards and for larger workloads you could choose 64 or 128. This means that you have the leeway to scale from 32 to 128 worker machines.
+
+In the :ref:`rt_blurb` use-case, shard count should be related to the total number of cores on the workers. To ensure maximum parallelism, you should create enough shards on each node such that there is at least one shard per CPU core. We typically recommend creating a high number of initial shards, e.g. 2x or 4x the number of current CPU cores. This allows for future scaling if you add more workers and CPU cores.
```

Comment:
This info is good, but could you also link to the production sizing section (e.g. for more information see ...)?

Reply:
Good idea. Done.